web-console/knikubevirt/import-vm/import-vm.md

```diff
@@ -0,0 +1,66 @@
+# Import VM
+The `Import virtual machine` action will be available in the `Create virtual machine` dropdown.
+
+It will also be in the empty state as a secondary action.
+
+![Import VM](img/import-vm.png)
+
+The Import virtual machine flow is very much the same as the Create a virtual machine flow with the exception of the general step.
+
+![General step](img/Step-1-basic-import-0.png)
+
+## Selecting provider 
+The Provider is chosen (in this case VMware is the only one available).
+
+![provider](img/Step-1-basic-import-1.png)
+
+## Selecting vCenter instance
+Once VMware is selected a new `vCenter instance` field appears. The user clicks it to see a list of previously-connected vCenter instances that have been stored as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/). The user selects one.
+
+![vCenter provider](img/Step-1-basic-import-2.png)
+
+## Selecting virtual machine
+When a vCenter instance is selected, The `Virtual Machine to import` dropdown is available with all the available VMs.
+
+![selected virtual machine](img/Step-1-basic-import-3.png)
+
+## vCenter connection error
+If the wizard cannot connect to the vCenter instance using the credentials stored within its Secret, an error message will be displayed. If the credentials are incorrect they can update them. If they check `update vCenter credentials secret` the action will be `check and update`. This should be the default because they have already created this secret. This allows them to update the secret without having to leave this flow.
+
+![Updating vCenter credentials](img/Step-1-basic-import-4.png)
+
+![checking vCenter credentials](img/Step-1-basic-import-4.2.png)
+
+If they leave the checkbox un-checked the action will just be `check`.
+
+![check vCenter credentials](img/Step-1-basic-import-4.1.png)
+
+## Connecting to new instance
+The user can also connect to a new vCenter instance by clicking `Connect to new instance` from within the dropdown.
+
+![connecting to new instance](img/Step-1-basic-import-5.png)
+
+New fields for vCenter URL, vCenter Username, vCenter Password, and VM to import appear along with a checkbox to `Save vCenter credentials secret`. This would be creating a new secret so it would be unchecked by default.
+
+The vCenter URL field should include a syntax hint with the desired formatting. The user is likely to paste a variety of URLs, so any FQDN the user submits should be validated and automatically corrected before being used to communicate with vCenter’s API.
+
+![check connection](img/Step-1-basic-import-6.png)
+
+When the user clicks the “Check” button a spinner icon appears within the button to indicate that the credentials are being checked.
+
+![checking connection](img/Step-1-basic-import-7.png)
+
+### Save credentials 
+The `Save vCenter credentials secret` checkbox becomes enabled to allow the user to save them as a new Kubernetes Secret.
```

Comment:
@andybraren have a look at this now and let me know what you think. I am thinking include the "secret" concept inside the label rather than hiding it in a tooltip.

Reply:
If we go with the behavior where a successful "Check and save" would basically collapse the fields into a newly-born "vCenter instance", then labeling the checkbox "Save as new vCenter instance" might make that behavior more expected.

I see the appeal of mentioning the word "secret" for the reasons discussed in that thread though. If we want to avoid a tooltip, maybe "Save as new vCenter instance secret?" would be an okay compromise?